package.json

```diff
@@ -33,11 +33,20 @@
 		"semver": "^6.3.0"
 	},
 	"devDependencies": {
+		"codecov": "^3.6.1",
```

Comment:
No changes need to be introduced for Codecov at this point. I'll handle that.

Reply:
Ah.  ok.